scripts/procedures/setConfig.ts

```diff
@@ -295,6 +295,9 @@ function configMaker(alias: Alias, config: SetConfig) {
   const enableCLNRestPlugin = config.advanced.plugins.clnrest
     ? "clnrest-port=3010\nclnrest-host=0.0.0.0\n"
     : "";
+  const enableCLNWebsocket = config.advanced.clams_remote_websocket
```

Comment:
Core Lightning can expose multiple ports for websocket connections. That said, I would be more specific in this const declaration so it is clear what it relates to i.e. `enableClamsWebsocket`.

Reply:
Agree, changed to 'enableClamsRemoteWebsocket'.